src/internal/connector/onedrive/api/drive.go

```diff
@@ -203,75 +203,6 @@ func (p *siteDrivePager) ValuesIn(l api.PageLinker) ([]models.Driveable, error)
 	return getValues[models.Driveable](l)
 }
 
-// GetDriveIDByName is a helper function to retrieve the M365ID of a site drive.
-// Returns "" if the folder is not within the drive.
-// Dependency: Requires "name" and "id" to be part of the given options
-func (p *siteDrivePager) GetDriveIDByName(ctx context.Context, driveName string) (string, error) {
-	var empty string
-
-	for {
-		resp, err := p.builder.Get(ctx, p.options)
-		if err != nil {
-			return empty, graph.Stack(ctx, err)
-		}
-
-		for _, entry := range resp.GetValue() {
-			if ptr.Val(entry.GetName()) == driveName {
-				return ptr.Val(entry.GetId()), nil
-			}
-		}
-
-		link, ok := ptr.ValOK(resp.GetOdataNextLink())
-		if !ok {
-			break
-		}
-
-		p.builder = mssites.NewItemDrivesRequestBuilder(link, p.gs.Adapter())
-	}
-
-	return empty, nil
-}
-
-// GetFolderIDByName is a helper function to retrieve the M365ID of a folder within a site document library.
-// Returns "" if the folder is not within the drive
-func (p *siteDrivePager) GetFolderIDByName(ctx context.Context, driveID, folderName string) (string, error) {
```

Comment:
This was part of a post-test cleanup, where the intent was to immediately delete the folders generated by the test restore.  The intent was good, but the implementation had some problems: it didn't support multi-drive deletion (drives outside of the standard Documents weren't getting cleaned up), and it was request heavy, which is not something we need more of in CI.  Since we have the cleanup script running, and we aren't using these calls elsewhere, there's no need to keep them around.